lib/teaspoon/configuration.rb

```diff
@@ -97,6 +96,12 @@ def use_framework(name, version = nil)
       def hook(group = :default, &block)
         @hooks[group.to_s] << block
       end
+
+      def no_coverage(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `no_coverage(*)` if you want the method to accept any arguments but don't care about them.